src/Halogen/Driver.purs

```diff
@@ -131,7 +131,7 @@ runUI c s element = _.driver <$> do
         when (isJust rp) $ render ref
         putVar rpRef Nothing
         q
-      HaltHF -> empty
+      HaltHF msg -> throwError $ error msg
```

Comment:
Hmm, I can indeed see how that would result in this happening. I don't know how to fix it though - and if it's currently happening, doesn't that mean SlamData is already broken?